assets/js/lib/jquery/dist/jquery.js

```diff
@@ -1,10346 +0,0 @@
-/*!
- * jQuery JavaScript Library v1.11.2
```

Comment:
I've never used Jekyll, but is there not a configuration file that specifies the dependencies (and their respective versions) for the application?

Reply:
Jekyll is purely server-side - it isn't opinionated in front-end dependencies. 

This project _did_ use bower for front-end package management (inherited from the forked project). Bower is functionally deprecated and not even very useful in the simplified scope of the current project. After this is resolved (for security), the next challenge is moving to a more future-proof package management.

Reply:
Okay that makes more sense - I saw the bower config but didn't see it updated here so I wasn't sure. Do we have an issue opened to address package management?

Reply:
Good idea. I added #221